.github/workflows/ci.yml

```diff
@@ -83,11 +83,14 @@ jobs:
                 run: |
                     cd build-release-g++
                     ctest -VV
-    build-fedora-32:
-        name: fedora-32
+    build-fedora:
+        name: Build on fedora
         runs-on: ubuntu-latest
+        strategy:
+            matrix:
+                fedora_version: [36, 37, 38, 39]
```

Comment:
https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#expanding-or-adding-matrix-configurations

It looks like you could include builds with or without the external libs.  I'd prefer to test both parts of we can.  Right now this change means none of the fedora builds will be tested with the embedded vendor libs.